server/models/PetType.js

```diff
@@ -0,0 +1,51 @@
+import pg from "pg"
+
+const pool = new pg.Pool({ connectionString: "postgres://postgres:password@localhost:5432/new_family_member_development" })
+
+class PetType {
+    constructor({ id, type, img_url, imgUrl, description }) {
+        this.id = id
+        this.type = type
+        this.imgUrl = imgUrl || img_url
+        this.description = description
+    }
+    static async findAll() {
+        try {
+            const result = await pool.query("SELECT * FROM pet_types;")
+            const typesData = result.rows
+            const types = typesData.map(type => new this(type))
+
+            return types
+
+        } catch (error) {
+            console.log(error)
+            throw (error)
+        }
+    }
+
+    static async findByType(type) {
+        const petFile = await import('./Pet.js')
+        const Pet = petFile.default
+        let petTypeId
+        if (type === "cats") {
+            petTypeId = 1
+        } if (type === "dogs") {
+            petTypeId = 2
```

Comment:
We should never hardcode ids into our code -- we don't know for sure if our database will always look like that!

Instead, update this code to query the database by type and find the record itself in your database, then read its id.

We're also running into an issue where conventionally, based on name, I'd expect this method to return a `PetType` object, but instead, it's returning a list of `Pet`s.

OPTION 1:
1. Update this method to query the `pet_types` table by type and find the record itself in your database, then read its id.
2. Rename the method `findPetsByType` to be clear that it's returning `Pet` objects.

OPTION 2: Even better: we could change this method to be more conventional by doing the following:
1. `findByType` should return a `PetType` object itself. Query the db by `type` column and return the PetType, much like `findById` would work
2. Then, create a `.pets()` association method which is an _instance_ method rather than a static method, and returns the `Pet`s for a given `PetType`